### DISCUSSION TITLE
HW1 - numpy, collections, vocab set

### DISCUSSION THREAD

**Discussion by nahyeonkaty** [seominjoon/kaist-ai605#13] [Q&A] [answered]
Good evening professor, I have two questions to ask!

1) Are we allowed to use numpy, collections?
2) When training, do we need to train with vocab set with words that occurred at least two times in train.tsv, or the whole vocab set(including the ones that occurred less than two times)?

Thank you!

**Comment by seominjoon** [comment, marked as answer]
1. Numpy and collections are fine.
2. Either is fine but at least two times is recommended. (in the real world, this is rather a design choice)
